examples/cpp/threading.cpp

```diff
@@ -0,0 +1,88 @@
+// :replace-start: {
+//   "terms": {
+//     "ThreadingItem_": ""
```

Comment:
Ah, I thought it was odd that this naming was in the generated code example - looks like we need to tweak the `replace` here and then regenerate the Bluehawk snippet.

```suggestion
//     "ThreadingExample_": ""
```

Reply:
Blar I was messing with this and forgot to fix it